### DISCUSSION TITLE
An error from skywalking agent 8.6.0 use jdk17

### DISCUSSION THREAD

**Discussion by yaboofzu** [apache/skywalking#10820] [Q&A] [answered]
The skywalking-agent version is: 8.6.0，
The  bytebuddy version is 1.11.18  ,  refer to skywalking change log：Support JDK 8-17 through upgrading byte-buddy to 1.11.18.
JDK version: OpenJDK version "17.0.3" 
plugin 使用了8.10.0 中的  bootstrap/jdk-threadpool-plugin

If agent.is_cache_enhanced_class=true, the following exception is reported:

<img width="1329" alt="image" src="https://github.com/apache/skywalking/assets/31541832/d84dd12d-dd15-40bb-8bab-302a3dad515f">

If agent.is_cache_enhanced_class=false , will not exception

**Comment by yaboofzu** [comment]
The above problems did not appear in the 1.8 version of jdk, but after I upgraded the JDK to 17, the above problems appeared. In addition, I used the agent of the official 8.10.0 and did not find the above problems. Therefore, I would like to know what changes have been made in addition to upgrading bytebuddy to support jdk17 for skywalkingagent.
上述问题，在jdk1.8 的版本中没有出现，但是我将jdk升级到17 后就出现了如上异常，另外我使用官方8.10.0 的agent  也没有发现上述异常。  因此我想了解官方为支持skywalkingagent 使用jdk17， 除了升级bytebuddy外还做了那些修改。

**Comment by wu-sheng** [comment, marked as answer]
We don't have suggestions about this. Check changelog by yourself, or update to the latest.

**Reply by yaboofzu** [reply]
We have made some custom changes in 8.6.0, and upgrading to the latest version will be a big change for us,
In addition, I have seen the change logs from skywalking 8.6.0 to 8.10.0, except for the modification of bytebuddy, no related modification has been found, so please help to inform me.
我们在8.6.0的版本上做了大量的自定义修改，如果升级到最新的版本，对我们来说改动量比较大，
另外我看了skywalking 8.6.0到8.10.0 对应的修改记录，除了bytebuddy的修改外，没有发现关联的修改，所以非常希望能得到官方的支持。

**Reply by wu-sheng** [reply]
What you did is your responsibility. That is the simple and straightforward answer from upstream.
